README.rst

```diff
@@ -1,61 +1,66 @@
+(It is used in some of the `prebuilt SageMaker Docker images for inference <https://docs.aws.amazon.com/sagemaker/latest/dg/pre-built-containers-frameworks-deep-learning.html>`__.)
 
-.. code:: python
+For more background information, please consult the Amazon SageMaker Developer Guide sections on `building your own container with Multi Model Server <https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html>`__ and `using your own models <https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms.html>`__.
 
-    from sagemaker_inference import model_server
+Installation
+------------
 
-    model_server.start_model_server(handler_service=HANDLER_SERVICE)
+To install this library in your Docker container, add the following line to your `Dockerfile <https://docs.docker.com/engine/reference/builder/>`__:
 
-The ``HANDLER_SERVICE`` is a string literal that points to the Python path of a Python file that will be executed by the
-model server for incoming invocation requests. This Python script is responsible for handling incoming data and passing it on to the engine for inference.
-The Python file should define a ``handle`` method that acts as an entry point for execution, this function will be invoked by the model server on a inference request.
+.. code:: dockerfile
 
-For more information on how to define your ``HANDLER_SERVICE`` file, see `Custom Service <https://github.com/awslabs/mxnet-model-server/blob/master/docs/custom_service.md>`__.
+    RUN pip3 install multi-model-server sagemaker-inference-toolkit
 
-Running tests
--------------
+`Here is an example <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/multi_model_bring_your_own/container/Dockerfile>`__ of a Dockerfile that installs SageMaker Inference Toolkit.
 
-To run all tests:
+Usage
+-----
 
-::
+To use the SageMaker Inference Toolkit, you need to do the following:
 
-    tox
-
-Using SageMaker Inference Toolkit
----------------------------------
-
-SageMaker Inference Toolkit serving stack leverages `Model Server for Apache MXNet (MMS) <https://github.com/awslabs/mxnet-model-server>`_ to server deep learning models trained using any ML/DL framework in SageMaker. Any SageMaker container can use the SageMaker Inference Toolkit to implement their serving stack. To use the Inference Toolkit, customers need to implement the following the components:
-
-- An inference handler responsible to load the model, and provide default input, predict, and output functions:
+1. Implement an inference handler, which is responsible for loading the model and providing input, predict, and output functions.
+   (`Here is an example <https://github.com/aws/sagemaker-pytorch-serving-container/blob/master/src/sagemaker_pytorch_serving_container/default_inference_handler.py>`__ of an inference handler.)
 
 .. code:: python
```

Comment:
a few comments on the code sample (can't comment on the lines directly because GH doesn't think they're close enough to lines you've changed):
* the docstrings in the code sample are missing newlines between sections.
* I think it could be nice to have a `default_model_fn` implementation in the example.
* `VALID_CONTENT_TYPES` isn't used

Reply:
Good catch. I'll update the code sample in a separate PR.